src/go/k8s/README.md

```diff
@@ -18,6 +18,7 @@ Official Kubernetes quick start documentation can be found at
 * Kubernetes 1.16 or newer
 * kubectl 1.16 or newer
 * kustomize v3.8.7 or newer
+* cert-manager v1.0.0 or newer
```

Comment:
I thought we are using v1.2.0, i.e. not testing with a lower version?

Reply:
I changed this based on Alena comment https://github.com/vectorizedio/redpanda/pull/707#discussion_r584490392

Reply:
I will resolve this comment as there is no respond.